app/src/main/java/org/mozilla/fenix/settings/AboutPage.kt

```diff
@@ -36,10 +38,13 @@ object AboutPage {
 
         substitutionMap["%build-date%"] = BUILD_DATE
 
-        context.resources.getString(R.string.about_content, appName).also { content ->
+        context.resources.getString(R.string.about_content, appName, SupportUtils.MOZILLA_MANIFSETO_URL)
+            .also { content ->
             substitutionMap["%about-content%"] = content
         }
 
+        substitutionMap["%wordmark%"] = BitmapFactory.decodeResource(context.resources,
```

Comment:
This should probably be run on a background thread for performance. ImageDecoder is the new and better way to do this, but I'm not sure they've added it to the support library yet. If they haven't, it's not useful as it's only for Pie.

Reply:
I'm still unsure why this is taking so long to load such a small image. Are we sure that loading a local asset that takes 5 seconds is better than just grabbing it from a hosted location?